modules/ossm-troubleshooting-sidecars.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/-ossm-troubleshooting-istio.adoc
+
+[id="ossm-troubleshooting-sidecars_{context}"]
+= Troubleshooting sidecars
+
+This topic describes common issues experienced with regards to sidecar initialization.
+
+== istio-init container fails to start
+The istio-init container fails to start when Istio is trying to create the sidecar.  The istio-init container logs show the following message:
+
+`iptables v1.6.0: can't initialize iptables table `nat': Table does not exist (do you need to insmod?) Perhaps iptables or your kernel needs to be upgraded.``
+
+With the release of Red Hat Enterprise Linux (RHEL) 8, the firewall has been migrated from iptables to nftables. The current default {ProductName} images are designed on top of RHEL 8 and are no longer compatible with iptables based systems. If the host operating system is based on iptables rather than nftables, the proxy-init image must be set to the CentOS 7 version instead.
+
+To fix this issue, add the following change to your `ServiceMeshControlPlane` and redeploy your pod.
+
+[source,yaml]
+----
+  istio:
+    global:
+      proxy_init:
+        image: proxy-init-centos7
```

Comment:
I'm not sure whether or not we officially support using the init container.  When using injection, we only support using cni (i.e. no init container).

Reply:
This was actually something I pulled from the Maistra web site before @brian-avery converted it to mirror the product docs.  I wasn't sure how old it was or if it was still relevant.  Brian, any idea?

Reply:
I don't think we support using the init container anymore. We supported OpenShift 3.11/OKD concurrently with 4.1 for a little while and this was a crutch in the mean time.

Reply:
Sounds like I can pull this topic out then.